Automattic/wp-calypso
# Jetpack plans: Add priority support to jetpack personal plan features

this PR just adds 'priority support' as an item in the personal plans list of features:
![image](https://cloud.githubusercontent.com/assets/1554855/21525592/90ac1476-cd1e-11e6-9885-c092c6ced19c.png)

To test:
=====
1. Go to https://calypso.live/jetpack/connect/vaultpress?branch=add/jetpack_plans_support and make sure that personal plan features include 'priority support', both in the yearly and monthly versions
